docs/samples/explanation/aix/mnist/rfserver/rfserver/__main__.py

```diff
@@ -27,6 +28,7 @@
 args, _ = parser.parse_known_args()
 
 if __name__ == "__main__":
+    logging.configure_logging(args.log_config_file)
```

Comment:
Should this be part of `kserve.logger` and pre-configured?

Reply:
After reading the discussion [here](https://github.com/kserve/kserve/pull/2884#discussion_r1190296386), I thought it is better not to configure logger at module level.

Reply:
do we have a example of how this config file looks like?

Reply:
@sivanantha321 Can you help add an example? Let's also document the behavior if `configure_logging` is not called

Reply:
https://github.com/kserve/website/pull/360